job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/VersionNumberContext.groovy

```diff
@@ -0,0 +1,62 @@
+package javaposse.jobdsl.dsl.helpers.wrapper
+
+import javaposse.jobdsl.dsl.Context
+
+class VersionNumberContext implements Context {
+
+    String environmentVariableName
+    String versionNumberString
+    String environmentPrefixVariable
+    boolean skipFailedBuilds
+    boolean useAsBuildDisplayName
```

Comment:
Please those variable names don't need to match with the xml configuration file. They should be more descriptive. for instance _useAsBuildDisplayName_ could be renamed to _displayBuildName_